resources/vlanresource_test.go

```diff
@@ -39,25 +39,25 @@ type vlanRsrcValidator struct {
 func (vt *vlanRsrcValidator) nextCfgState() {
 	vt.expCfg = vt.expCfg[1:]
 	if len(vt.expCfg) > 0 {
-		log.Printf("after pop cfg is: %+v\n", vt.expCfg[0])
+		log.Debugf("after pop cfg is: %+v\n", vt.expCfg[0])
```

Comment:
@mapuri - are these excessive debugs? Perhaps the dev level debug should be decprecated after some code solidification and add others that might give events and gives a flow of an operation